src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
+@import 'variables';
+
 body {
+  width: 100vw;
+  height: 100vh;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: ($square-size * 7) + ($days-gap * 6);
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  padding: $calendar-padding;
+
+  @each $day in $days {
+    &--start-day-#{$day} > .calendar__day:first-child {
+      @if $day == tue {
+        margin-left: $square-size + $days-gap;
+      } @else if $day == wed {
+        margin-left: 2 * ($square-size + $days-gap);
+      } @else if $day == thu {
+        margin-left: 3 * ($square-size + $days-gap);
+      } @else if $day == fri {
+        margin-left: 4 * ($square-size + $days-gap);
+      } @else if $day == sat {
+        margin-left: 5 * ($square-size + $days-gap);
+      } @else if $day == sun {
+        margin-left: 6 * ($square-size + $days-gap);
```

Comment:
simplify it and just move those numbers to days var

```
$days: (
  'mon': 0,
  'tue': 1,
  'wed': 2,
  'thu': 3,
  'fri': 4,
  'sat': 5,
  'sun': 6,
);
```

Reply:
Thanks, solved.